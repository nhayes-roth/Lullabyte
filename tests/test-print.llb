void main(){ 
	print(19);
	print(1111.111);
}