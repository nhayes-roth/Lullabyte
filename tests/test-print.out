19
1111.111
